examples/FileIO/WriteBytesToFile.dfy.expect

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
This file also doesn't have particularly interesting content, so we could just check for successful execution.

Reply:
And while I was writing this you updated it. :)